src/matlab2tikz.m

```diff
@@ -3530,18 +3523,17 @@ function warnFacetedInterp(m2t)
     if strcmpi(getEnvironment(), 'Octave')
         sData = sData.^2/2;
     end
-    sData = translateMarkerSize(m2t, matlabMarker, sqrt(sData)/2);
+    sData = translateMarkerSize(m2t, rawMarker.style, sqrt(sData)/2);
 
     drawOptions = opts_new();
     if length(cData) == 3
         [m2t, drawOptions] = getScatterOptsOneColor(m2t, h, drawOptions, ...
-                                                markOptions, tikzMarker, ...
-                                                cData, sData, constMarkerkSize);
+                                                rawMarker, cData, sData, ...
+                                                constMarkerkSize);
```

Comment:
I don't think it's constant all the time: `sData` could be a vector.
But you do have a point that it could be calculated from `sData` again.